content/download/debian.adoc

```diff
@@ -6,63 +6,194 @@ weight = 10
+----
+$ sudo apt update
+----
+
+For more information about backports you might want to visit the Debian Wiki
+site about https://wiki.debian.org/Backports[Backports] and / or also the
+https://backports.debian.org/Instructions/[Backports website].
+
+=== Other KiCad related packages and their relations
+
+For more flexibility the various KiCad packages are provided by also various
+binary packages you might want need to install beside the `kicad` main package
+that is containing the KiCad applications. For example there are existing
+dedicated packages for the footprints, symbols, templates and 3D-models and
+also KiCad documentation related packages which aren't necessarily
+automatically get installed (depends on the configuration of your system) if
+you install the `kicad` package.
+
+If you are looking for all KiCad related packages you can search for them
+simply by your preferred package manager tool, e.g. by `apt`.
 
 [source,bash]
-sudo apt update
-sudo apt install kicad
+----
+$ sudo apt search kicad*
+----
+
+For a useful working with KiCad you will need to have installed the following
+packages: `kicad` and `kicad-libraries` and sometimes also the package
+`kicad-demos`.
+Typically these packages get installed automatically if you haven't changed the
+system default on installing of recommended packages. Please don't install the
+package `kicad-common` directly, it's a transitional package and will go away,
+it can get removed if you've installed this.
+
+You can get more information about the relation between the various `kicad`
+packages in Debian please have a look at a graphic on the
+https://wiki.debian.org/KiCad[Debian wiki page] for KiCad packages.
+
+'''
+
+=== KiCad packages within the Debian releases
+
+==== Debian Old-Old-Stable (Codename Jessie)
+
+WARNING: The Jessie release isn't supported any more and wont get updates, please
+upgrade your system to at least Debian Stretch!
+
+'''
+
+==== Debian Old-Stable (Codename Stretch)
+
+===== https://packages.debian.org/stretch-backports-sloppy/kicad[*Old-Stable* (Backports)]
+
+Version: 5.1.4
+
+====== Installation
+
+[source.bash]
+----
+$ sudo apt install -t stretch-backports kicad
+----
+
+Please ensure you have updated the local database of the package management
+system before installing!
+
+TIP: Keep in mind that this version can't provide all functions of KiCad as
+the required packages for this are too old in Stretch or not available!
+Especially the support for Python scripting is limited to Python 2! Some
+external AddOns wont work.
+
+'''
+
+===== https://packages.debian.org/stretch/kicad[*Old-Stable* (Release)]
+
+Version: {{< repology debian_oldstable >}}
 
-Offline docs are available in separate packages named for example
-`kicad-doc-en`. You can search for them with _apt_ or _apt-cache_ for example.
+====== Installation
 
 [source.bash]
-apt search kicad-doc
-# or alternatively if you are on wheezy
-apt-cache search kicad-doc
+----
+$ sudo apt install kicad
```

Comment:
I think we should remove the dollarsign from the commands, it is annoying to copy.

Reply:
I'm personally against this, as long as I know about such copy ready cli text or the places there I work on this the dollar sign is used here. It shows the shell is a non root shell aka a "normal" shell there the user is working on. Contrary if the number sign `#` is used to mark a root shell.

But I can live with dropping the dollar sign here. Changed within next force-push.